x/claim/client/cli/query.go

```diff
@@ -0,0 +1,33 @@
+package cli
+
+import (
+	"fmt"
+	// "strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	// "github.com/cosmos/cosmos-sdk/client/flags"
+	// sdk "github.com/cosmos/cosmos-sdk/types"
```

Comment:
can we please remove these unused import?

Reply:
[695aa26](https://github.com/arkeonetwork/arkeo/pull/29/commits/695aa2671d3af12adf78bd728b14aafa1398889e)